aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -1,4 +1,75 @@
 module aptos_framework::transaction_context {
+
+    use std::features;
+
+    /// AUID feature is not supported.
+    const EAUID_NOT_SUPPORTED: u64 = 1;
+
+    /// A wrapper denoting aptos unique identifer (AUID)
+    /// for storing an address
+    struct AUID has drop, store {
+        unique_address: address
+    }
+
+    /// Return the transaction hash of the current transaction
+    public native fun get_txn_hash(): vector<u8>;
```

Comment:
even though we are not using it, does this need to be gated?

Reply:
I couldn't really think of a reason to gate it.